RELEASE-NOTES.txt

```diff
@@ -1,3 +1,10 @@
+1.10.0
+------
+* Adding a block from the post title now shows the add block here indicator.
+* Deselect post title any time a block is added
+* Fix merging of empty paragraph blocks when added from a post's title
+* Fix loss of center alignment in image captions
```

Comment:
Looking back at this now, seems like I perhaps should have called this out as being an Android-specific issue. I'm happy to add that change if you agree.

Reply:
Ok, I'll update and remove it from https://github.com/wordpress-mobile/WordPress-iOS/pull/12210/files#diff-a3d054590c32de57d87be233927e755dR9